-/config.php

```diff
@@ -7,12 +7,12 @@
  */
 // LOGIN
 define('USERNAME', 'admin');
-define('PASSWORD', 'cschirmer');
+define('PASSWORD', '');
 
 // DATABASE
 define('DB_NAME', 'lessn');
 define('DB_USERNAME', 'root');
-define('DB_PASSWORD', 'cschirmer');
+define('DB_PASSWORD', '');
```

Comment:
Perhaps it would be good to set it up to allow the password to be injected via env var, or a non-versioned config file?

Reply:
Yes, you have right. My local settings was the last hint to me to change them.